bucket/revouninstaller.json

```diff
@@ -6,21 +6,15 @@
     "url": "https://www.revouninstaller.com/download-free-portable.php#/dl.zip",
     "hash": "f543d9e0627f4170674310f5759498bc65a08446c3c2693f9771493c9d3063f5",
     "extract_dir": "RevoUninstaller_Portable",
-    "installer": {
-        "script": [
-            "function CreateFile([String] $file, [String] $content = $null) {",
-            "    if (-not (Test-Path \"$persist_dir\\$file\")) {",
-            "        Write-Host 'File' $file 'does not exists. Creating.' -f Yellow",
-            "        Set-Content \"$dir\\$file\" $content -Encoding Ascii",
-            "    }",
-            "}",
-            "CreateFile \"ctrlbars.dat\"",
-            "CreateFile \"un_report.dat\"",
-            "CreateFile \"winapp_report.dat\"",
-            "CreateFile \"settings.ini\""
-        ]
-    },
-    "post_install": "(Get-Content \"$dir\\settings.ini\") -replace 'AU on startup=1','AU on startup=0' | Set-Content \"$dir\\settings.ini\"",
+    "pre_install": [
+        "'ctrlbars.dat', 'un_report.dat', 'settings.ini', 'winapp_report.dat' | ForEach-Object {",
+        "    if (!(Test-Path \"$persist_dir\\$_\")) {",
+        "        Write-Host 'File' $_ 'does not exists. Creating.' -f Yellow",
+        "        Set-Content \"$dir\\$_\" $content -Encoding Ascii",
+        "    }",
+        "}",
+        "(Get-Content \"$dir\\settings.ini\") -replace 'AU on startup=1','AU on startup=0' | Set-Content \"$dir\\settings.ini\""
```

Comment:
DRY.

- Just save the AU into regex capture group and then reference it in replace
- Always use encoding ascii in set-content